### PR TITLE
Issue #286 Default values and update passing of configuration to service,

### PR DESCRIPTION
**Pull request by darranl**

### CODE REVIEW

**Review by hpehl — APPROVED** [review]
As for the previous PRs: My knowledge about the subsystem API is limited. But from your comments and code: LGTM.

File: subsystem/src/main/java/org/wildfly/extension/grpc/GrpcSubsystemAdd.java [hunk 1]
```diff
@@ -59,88 +55,62 @@ protected void performBoottime(OperationContext context, ModelNode operation, Mo
...
+        configuration.setHandshakeTimeout(GrpcSubsystemDefinition.GRPC_HANDSHAKE_TIMEOUT.resolveModelAttribute(context, model)
+                .asInt());
+        configuration.setInitialFlowControlWindow(
+                GrpcSubsystemDefinition.GRPC_INITIAL_FLOW_CONTROL_WINDOW.resolveModelAttribute(context, model)
+                        .asInt());
+        configuration
+                .setKeepLiveTime(isDefined(GrpcSubsystemDefinition.GRPC_KEEP_ALIVE_TIME, model)
+                        ? GrpcSubsystemDefinition.GRPC_KEEP_ALIVE_TIME.resolveModelAttribute(context, model)
+                                .asLong()
+                        : -1);
```
Comment (jamezp) [inline subsystem/src/main/java/org/wildfly/extension/grpc/GrpcSubsystemAdd.java:76]: Not critical, but these could all be `.asInt(-1)` where `-1` is the default.

**Review by jamezp — APPROVED** [review]
LGTM. I've made one minor comment, but it's not critical.

**Review by jamezp — APPROVED** [review]
LGTM

**Pull request merged**